docusaurus.config.js

```diff
@@ -47,10 +47,17 @@ module.exports = {
         sidebarPath: require.resolve("./optimize_sidebars.js"),
         editUrl: "https://github.com/camunda/camunda-docs/edit/main/",
         versions: {
+          "3.11.0": {
+            label: "8.3 / 3.11.0",
+          },
+          "3.10.0": {
+            banner: "none",
+          },
           "3.9.0": {
             banner: "none",
           },
-          "3.8.0": {
+          // surprising, yes, but true: 3.8 should show unsupported banner, but 3.7 should not.
+          "3.7.0": {
             banner: "none",
           },
```

Comment:
3.7.0 remains supported, [as discussed in Slack](https://camunda.slack.com/archives/C55U06YRG/p1696454669814269?thread_ts=1696454303.091489&cid=C55U06YRG).